src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/content/components/__tests__/MethodologyNotesSection.test.tsx

```diff
@@ -107,6 +107,44 @@ describe('MethodologyNotesSection', () => {
         screen.getByRole('button', { name: 'Add note' }),
       ).toBeInTheDocument();
     });
+
+    test('renders correctly when there are no notes', () => {
+      render(
+        <EditingContextProvider initialEditingMode="edit">
+          <MethodologyNotesSection
+            methodology={testMethodologyContentWithoutNotes}
+          />
+        </EditingContextProvider>,
+      );
+      expect(screen.getByText('Last updated')).toBeInTheDocument();
+      expect(screen.getByTestId('Last updated-value')).toHaveTextContent('TBA');
+      expect(
+        screen.getByRole('button', { name: 'Add note' }),
+      ).toBeInTheDocument();
+    });
+
+    test('shows the most recent date for Last Updated', () => {
+      const oldNote = {
+        id: 'note-old',
+        content: 'Note Old',
+        displayDate: new Date('2020-01-01T00:00:00'),
+      };
+      const testMethodologyContentWithOldNote = {
+        ...testMethodologyContentWithNotes,
+        notes: [oldNote, ...testMethodologyContentWithNotes.notes],
+      };
```

Comment:
Could add some explicit types for these variables

Reply:
sorted 👍